code/datums/storage/storage.dm

```diff
@@ -1072,11 +1072,11 @@ GLOBAL_LIST_EMPTY(cached_storage_typecaches)
 	collection_mode = (collection_mode+1)%3
 	switch(collection_mode)
 		if(COLLECT_SAME)
-			to_chat(user, span_notice("[resolve_parent] now picks up all items of a single type at once."))
+			resolve_parent.balloon_alert(user, "single type")
 		if(COLLECT_EVERYTHING)
-			to_chat(user, span_notice("[resolve_parent] now picks up all items in a tile at once."))
+			resolve_parent.balloon_alert(user, "all on tile")
 		if(COLLECT_ONE)
-			to_chat(user, span_notice("[resolve_parent] now picks up one item at a time."))
+			resolve_parent.balloon_alert(user, "single item")
```

Comment:
```suggestion
			resolve_parent.balloon_alert(user, "will now pick up one at a time")
```

The existing text you had for these 3 was definitely short but would not be clear to me as to what they actually mean. These are longer but not too terse.